CHANGES

```diff
@@ -6,6 +6,15 @@ Tactics
 - New tactic "extensionality in H" which applies (possibly dependent)
   functional extensionality in H supposed to be a quantified equality
   until giving a bare equality.
+- New tactic "inversion_sigma" which turns equalities of dependent
+  pairs (e.g., "existT P x p = existT P y q", frequently left over by
+  "inversion" on a dependent type family) into pairs of equalities
+  (e.g., a hypothesis "H : x = y" and a hypothesis of type "rew H in p
+  = q"); these hypotheses can subsequently be simplified using
+  "subst", without ever invoking UIP / axiom K.  For more fine-grained
```

Comment:
There is the risk that UIP or axiom K does not ring a bell to basic users (so maybe expanding `without ever invoking any kind of axiom asserting uniqueness of identity proofs`? -- maybe not really clearer for the basic user, but at least less jargon).